examples/cfq/syntax-based/grammar.py

```diff
@@ -0,0 +1,83 @@
+import re
+
+GRAMMAR_STR = """
+  query: select_query
+  select_query: select_clause "WHERE" "{" where_clause "}"
+  select_clause: "SELECT" "DISTINCT" VAR 
```

Comment:
I think `VAR` is always `?x0` in this case.

```suggestion
  select_clause: "SELECT" "DISTINCT" "?x0"
```